CSC322-Grinnell/mayflower-directory
# Add tests for profiles import method and fixes redirect after profile edit

Resolves #211
Resolves #205

# Changes

Adds tests for the 'import' method of profiles
Changes a redirect 
# Testing

1. Git checkout small-things
2. Run `rails test`
3. Rails s -b $IP -p $PORT
4. Edit a profile in the directory, check that it redirects back to the profile you just edited
